### ISSUE TITLE
How about the performance compared with the original U-Net/nnUNet encoder?

### ISSUE THREAD

**Issue by CaiziLee** [Beckschen/TransUNet#34]

**Comment by lisherlock** [comment]
I have used transUnet to train the lung segmentation images from Kaggle, but I find the performance is similar to AttUnet. Maybe my hardware is poor to train it. :(

**Comment by lisherlock** [comment]
> I have used transUnet to train the lung segmentation images from Kaggle, but I find the performance is similar to AttUnet. Maybe my hardware is poor to train it. :(

Yep, I also train lung segmentation images from Kaggle, and i find the performance is not better than Unet. My hardware is 2080ti, so sad.

**Comment by andife** [comment]
Hi @lisherlock : Can you share your code? or do you have published for code at kaggle?
